mcs/class/WindowsBase/WindowsBase-net_4_x.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
How is it generated?
Should it be regenerated and commited after that instead or ignored/removed from git?